test/Conversion/TritonToStructured/triton-to-structured-prepass.mlir

```diff
@@ -197,3 +197,57 @@ module {
 
 // CHECK-COUNT-6: tts.get_structured_state
```

Comment:
Could we check there's no i1 tts.get_structured_state?
Current check only check number of tts.get_structured_state, not the type.

Reply:
Sorry. I looked wrong lines.